chapters/synchronous.tex

```diff
@@ -295,8 +295,8 @@ \subsection{Argument Restrictions (Component Expression)}\label{argument-restric
 \end{lstlisting}
 \end{example}
 
-Note that the operators defined in this chapter do not automatically vectorize,
-but some operate on arrays in a similar way.
+None that the operators defined in this chapter vectorize, but some can pass through an array of clocked expressions (but not of clocks) in a similar way.
+They are not callable in functions.
```

Comment:
Can clock or clocked expressions be used in functions in general?

Reply:
Clocked expressions can be sent to functions, but they behave as normal expressions; so nothing special there.
However, for Clocks it is less clear - and it makes sense to also forbid them in functions as they don't make sense.

Reply:
But I couldn't figure out a good formulation for that.

Reply:
I must say that I don't understand the where we fall short by saying that none of the operators are callable in functions.  What am I missing?

Reply:
I realized that you could write:
```
  function foo
    input Clock c;
....
end foo;
...foo(Clock(2));
```
So you don't have to call these operators in functions to have clocks inside them, and I don't see what Clocks mean inside functions.

However, we didn't forbid it before - so not a blocker.

Reply:
Oh, but that should be easy enough to address by not allowing `Clock` in function formal parameters, right?

Reply:
Yes, it would make sense to me to include the restriction in 12.2 where function restrictions are specified. I had checked there first before asking my question.

Reply:
Ok, added.